src/App.tsx

```diff
@@ -1,40 +1,45 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import { Tabs } from './components/Tabs';
 import './App.scss';
 
+export interface TabType {
+  id: string;
+  title: string;
+  content: string;
+}
+
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
 export const App: React.FC = () => {
+  const [activeTab, setActiveTab] = useState(tabs[0]);
```

Comment:
![image](https://github.com/mate-academy/react_tabs/assets/99403198/d27027af-fa00-411f-a59a-b019ba05e786)
```suggestion
  const [activeTabId, setActiveTabId] = useState(tabs[0].id);
```

Reply:
@devWizHub 
take attention to this comment. And make a little bit improvements.